Update Symfony Monolog integration (#3510)
getsentry/sentry-docs: src/platforms/php/guides/symfony/index.mdx

### Monolog Integration

> If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the normal error listener approach, you also need this additional configuration to log the errors correctly:

Reviewer: ```suggestion
If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the typical error listener approach, you need this additional configuration to log the errors correctly:
```

```yaml {filename:config/packages/sentry.yaml}
sentry:
    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry

monolog:
    handlers:
        sentry:
            type: sentry
            level: !php/const Monolog\Logger::ERROR
            hub_id: Sentry\State\HubInterface
```

> If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) before `3.7`, you need to

Reviewer 2: ```suggestion
If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) prior to `3.7`, you need to
```

> If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) below `3.7` you need to

Reviewer: "Below" sounded a bit awkward to me, but do what makes sense here.

```suggestion
If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) before `3.7`, you need to
```

Author: @imatwawana "before" sounds better, the reason I used "below" was cause that word is used on line 22.
